bc/subscription/tests/factories.py

```diff
@@ -13,13 +18,25 @@ class Meta:
     cl_docket_id = factory.LazyAttribute(
         lambda _: faker.random_int(100_000, 400_000)
     )
+    docket_name = factory.LazyFunction(local_faker.case_name)
+    docket_number = factory.LazyFunction(local_faker.docket_number)
+    court_name = factory.LazyAttribute(lambda _: local_faker.court_name())
 
     @factory.post_generation
     def channels(self, create, extracted, **kwargs):
+        """
+        Add channels to this subscription
+        @see https://factoryboy.readthedocs.io/en/stable/reference.html?highlight=post_generation#post-generation-hooks
+        @example:
+            some_channel = ChannelFactory.create()
+            SubscriptionFactory(channels=[channel_1, channel_2]) #
+            subscribes this to channel_1 and channel_2
+        """
         if not create or not extracted:
             return
-
+        print(f"extracted: {extracted}")
```

Comment:
I think we should remove this print statement.

Reply:
Oops! Yes, I definitely should.

Reply:
Thanks!